src/pages/security/static/_scope_content.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react'
```

Comment:
You don't need React for this file, we can also keep it as a `.ts`.

Reply:
we have Localize component here though, guess were going to need react imported

Reply:
We don't render the component out so it's fine. It's just a list of data.